src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,142 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final int TEST_AGE = 22;
+    private static final String DEFAULT_LOGIN = "defaultLogin";
+    private static final String TEST_LOGIN = "testLogin";
+    private static final String TEST_LOGIN_1 = "testLogin1";
+    private static final String TEST_PASSWORD = "testPassword";
+    private static RegistrationService service;
+
+    @BeforeAll
+    static void beforeAll() {
+        service = new RegistrationServiceImpl();
+    }
+
+    @Test
+    void register_nullLogin_notOk() {
+        assertThrows(RegistrationException.class,
+                () -> service.register(new User(null, TEST_PASSWORD, TEST_AGE)));
```

Comment:
Consider to create object as variable to pass it to the method to make the code more clear and readable